glidertools/mapping.py

```diff
@@ -1118,219 +1118,223 @@ def grid_flat_dataarray(xda, bins=None):
 
 try:
     import pykrige as pk
+except ImportError:
+    pk = None
```

Comment:
Could this not be moved into the function itself. 
And then raise it as an error rather than a warning, as the function will fail in any case. 
In the variogram function it might look like this

```python
try: 
    import pykrige as pk
except ImportError:
    raise ImportError('The variorum function requires that you install pyKrige, please do this manually')

Reply:
Some friends of mine from the Pangeo community convinced me that this is the more appropriate way to do it (I think this is also the stylistic choice made in xarray etc). Obviously, both cases work. I guess the cleaner thing about this is that the entire function is not sitting inside a try.